DEVELOPERS_README.dox

```diff
@@ -0,0 +1,180 @@
+
+/*! \page VOTCA Developers Guide
```

Comment:
The syntax is `\page NAME title ...`, so you created a `VOTCA` page? Maybe `dev_guide` would be better.